torchvision/prototype/datasets/_builtin/caltech.py

```diff
@@ -95,7 +95,7 @@ def _prepare_sample(
             bounding_box=BoundingBox(
                 ann["box_coord"].astype(np.int64).squeeze()[[2, 0, 3, 1]], format="xyxy", image_size=image.image_size
             ),
-            contour=_Feature(ann["obj_contour"].T),
```

Comment:
What is this change?

Reply:
It seems my IDE was overly ambitious when renaming variables. I'll revert this.